audit/schema.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE decrypt_event (
```

Comment:
would we need different tables for encrypt and decrypt, or just put everything into a single table named "events" ?
from a querying perspective, I think a single table would be fine, and just store the `action-{encrypt|decrypt}` as a column.

Reply:
I'm not sure if there's any practical difference. I think one table per event type is better because different events might include other data. If you need to query everything, you can always just join several tables.